README.md

```diff
@@ -89,7 +108,6 @@ npm test
 From within the root directory:
 
 ```sh
-npm install -g webpack
```

Comment:
Overall looks good, but I don't think the stuff about npm install is necessary for the documentation that's specifically about API routes.

Reply:
Since they told us to write API routes in readme, I just added API routes to readme,  and the rest is what already was there. I think we need npm stuff in readme so that others know how to open out project

Reply:
Ah ok, thought this would be a separate readme just for the API. That makes sense.